web/themes/contrib/civictheme/civictheme.post_update.php

```diff
@@ -794,3 +794,27 @@ function civictheme_post_update_alert_visibility_validation(): string {
   }
   return (string) (new TranslatableMarkup('Update to alert api view skipped, view does not exist.'));
 }
+
+/**
+ * Update the view mode 'civictheme_snippet' to 'civictheme_navigation_card'.
+ *
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+function civictheme_post_update_update_view_mode_civictheme_navigation_card_ref(): string {
+  // Get the config factory.
+  $config_factory = \Drupal::configFactory();
+
+  // Load the configuration.
+  $config = $config_factory->getEditable('core.entity_view_display.paragraph.civictheme_navigation_card_ref.default');
```

Comment:
@joshua-salsadigital Can you check to see if it exists - look at https://github.com/civictheme/monorepo-drupal/pull/1322 for guidance on how to check